spec/index.bs

```diff
@@ -3479,18 +3484,6 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
                                 with the data |bufferUpdate|.`data`.
```

Comment:
Here we are trying to update the content of the buffer. But what if the device is already lost? It isn't an error to call Unmap() even when the device is lost.